deltachat-ios/Controller/WelcomeViewController.swift

```diff
@@ -17,40 +17,19 @@ class WelcomeViewController: UIViewController, ProgressAlertHandler {
 
     private lazy var welcomeView: WelcomeContentView = {
         let view = WelcomeContentView()
-        view.onLogin = {
-            [unowned self] in
-            self.coordinator?.presentLogin()
+        view.onLogin = { [unowned self] in
+            self.coordinator?.showLogin()
         }
-        view.onScanQRCode = {
-            [unowned self] in
+        view.onScanQRCode  = { [unowned self] in
```

Comment:
i think, this reformatting (and the other ones, including whitespace changes) has nothing to do with the gist of this pr.

no need to change this now, however, for the future, please keep things in a pr more focused, esp. when a pr is about relatively complicated things as this one.